src/monitor/node_active_protocol.c

```diff
@@ -79,6 +79,8 @@ PG_FUNCTION_INFO_V1(synchronous_standby_names);
 Datum
 register_node(PG_FUNCTION_ARGS)
 {
+	checkPgAutoFailoverVersion();
```

Comment:
1. For consistency we should do this change as well for `get_nodes` and `get_other_nodes`. Although it doesn't parse arguments before the call, it's still nice to always do this check at the top of the function. Also to keep in mind future changes to the code. 
2. UDFs in `formation_metadata.c` don't have any version checks. I guess to be safe we should add them there at the top of each UDF as well.

Reply:
Done now, thanks!